sqlglot/generator.py

```diff
@@ -3625,13 +3625,15 @@ def merge_sql(self, expression: exp.Merge) -> str:
         using = f"USING {self.sql(expression, 'using')}"
         on = f"ON {self.sql(expression, 'on')}"
         expressions = self.expressions(expression, sep=" ", indent=False)
+        returning = self.sql(expression, "returning")
+        if returning:
```

Comment:
What we had before was somewhat consistent with how `RETURNING` for other SQL expressions worked, this makes it consistent. Returning would be a list of columns (for MERGE), take this example:

```
returning=[
    Anonymous(
      this=merge_action),
    Column(
      this=Star(),
      table=Identifier(this=w, quoted=False))])
```


While for other expressions such as delete/insert, it would be a `Returning` object with expressions:

```
returning=Returning(
  expressions=[
    Star()]))
```

This makes things consistent between INSERT/DELETE/MERGE.

Reply:
Seems like a reasonable refactor.